cpp/mrc/tests/test_mrc.hpp

```diff
@@ -26,13 +26,23 @@
 #include <chrono>
 #include <condition_variable>
 #include <cstddef>
+#include <cstdlib>
 #include <mutex>  // for mutex & unique_lock
 
 #define TEST_CLASS(name)                      \
     class Test##name : public ::testing::Test \
     {}
 
 namespace mrc {
+
+inline void skip_if_in_ci()
+{
+    if (std::getenv("CI") != nullptr)
+    {
+        GTEST_SKIP() << "Test skipped in CI";
+    }
+}
+
```

Comment:
Why is this a function and `SKIP_IF_IN_CI` one a macro?

Reply:
i forgot i had put it in test_mrc.hpp first, then realized we didn't need it in the public test.

Reply:
i added the updated macro to the public tests as well in case we need it.